docs/processors/getting-started.mdx

```diff
@@ -27,11 +27,11 @@ attached to a single parent, which can be either a connector or a pipeline:
 When it comes to using a processor, Conduit supports different types:
 
 - [Built-in processors](/docs/processors/builtin) will perform the most common operations you could expect such as filtering fields, replacing fields, posting payloads to a HTTP endpoint, etc. These are already coming as part of Conduit, and you can simply start using them with a bit of configuration. [Check out this document to see everything that's available](/docs/processors/builtin).
-- [Standalone processors](/docs/processors/standalone-processors) are the ones you could write yourself to do anything that's not already covered by the [Built-in](/docs/processors/builtin) ones. [Here's](/docs/processors/standalone-processors) more information about them.
+- [Standalone processors](/docs/processors/standalone are the ones you could write yourself to do anything that's not already covered by the [Built-in](/docs/processors/builtin) ones. [Here's](/docs/processors/standalone more information about them.
```

Comment:
The links for `/docs/processors/standalone` are broken (missing right parenthesis).